sippy-ng/src/prow_job_runs/ProwJobRun.js

```diff
@@ -102,7 +114,16 @@ export default function ProwJobRun(props) {
       })
       .catch((error) => {
         setFetchError(
-          'Could not retrieve intervals for ' + props.jobRunID + ', ' + error
+          'Could not retrieve intervals for ' +
+            'jobRunID=' +
+            props.jobRunID +
+            'jobName=' +
+            props.jobName +
+            'pullNumber=' +
+            props.pullNumber +
+            'repoInfo=' +
+            ', ' +
+            error
```

Comment:
i think you're lacking spaces between your params.

Reply:
fixed and squashed -- thx